src/components/EpicView/EpicView.tsx

```diff
@@ -62,6 +71,14 @@ export function EpicView() {
           </Group>
         </Group>
         <Title mb="sm">Epics</Title>
+        <TextInput
+          placeholder="Search by epic summary, id, creator, labels, status or assignee..."
```

Comment:
```suggestion
          placeholder="Search by summary, id, creator, labels, status or assignee..."
```
The user knows its about epics, since the whole view is about epics.